src/components/navbar.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { NavLink } from 'react-router-dom';
+
+import './component.css';
+
+const NavBar = () => (
+<nav><ul><li><NavLink>Home</NavLink></li></ul></nav>
+);
+
+export default NavBar;
```

Comment:
You have done an excellent job. However, you haven't installed 'react-router-dom' which is what <NavLink> is depending on. I will have to install it from my end. In all, this is a great start. Keep it up.